.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# Legal
+/LICENSE  @getsentry/owners-legal
+
+# Migrations
+south_migrations/  @getsentry/owners-migrations
+
+# Snuba
+/src/sentry/eventstream/        @getsentry/owners-snuba
+/src/sentry/utils/snuba.py      @getsentry/owners-snuba
+/src/sentry/tsdb/snuba.py       @getsentry/owners-snuba
+/src/sentry/tsdb/redissnuba.py  @getsentry/owners-snuba
+/src/sentry/search/snuba/       @getsentry/owners-snuba
+/src/sentry/tagstore/snuba/     @getsentry/owners-snuba
+
+# Ted stuff
+/src/sentry/similarity/              @tkaemming
+/src/sentry/receivers/similarity.py  @tkaemming
+*.lua                                @tkaemming
```

Comment:
I think some of these are trivial enough that it makes sense to not special case them to me — for example, the [lock release implementation](https://github.com/getsentry/sentry/blob/master/src/sentry/scripts/utils/locking/delete_lock.lua) or the [quota check script](https://github.com/getsentry/sentry/blob/master/src/sentry/scripts/quotas/is_rate_limited.lua) are pretty straightforward and I think most people could responsibly make changes without much help, if any.

It probably makes sense for things like digests, similarity, etc. though — not necessarily because they're Lua (though that's part of it), but because the specific implementations are generally part of a larger system that requires more background to work on.

Reply:
I just wanted to give you all Lua.

Reply:
I know. 😭